components/formats-gpl/src/loci/formats/in/DeltavisionReader.java

```diff
@@ -2659,9 +2659,40 @@ else if (lensID < 20000) {
         break;
       case 18107: // API 10X, BH
         lensNA = 0.15;
+
+        if (checkSuffix(currentId, "rcpnl")) {
```

Comment:
Do all the objective IDs defined below apply to files with both extensions or should they also only be mapped in the case of `rcpnl` files?

Reply:
I'm honestly not sure.  We don't have any examples of non-```rcpnl``` files with any of these IDs, but that's not necessarily conclusive.  It would be easy enough to make 18108-18111 only apply to ```rcpnl``` files, if that's preferable.